calico-enterprise/_includes/components/ReqsSys.js

```diff
@@ -29,9 +29,11 @@ function NodeRequirementsEnt(props) {
           <ul>
             {(props.orch === orchestrators.Kubernetes || props.orch === orchestrators.HostProtection) && (
               <>
-                <li>CentOS 8</li>
+                <li>Red Hat Enterprise Linux 7 and 8 or binary-compatible operating systems</li>
+                  <ul>
```

Comment:
I think we should remove this reference. SELinux isn't a requirement, but rather a special installation option for RHEL.